### ISSUE TITLE
GraphQL is not working on release-22.04 

### ISSUE THREAD

**Issue by bryanboza** [dotCMS/core#22013]
When you try to use the graphQL in the release branch, we are getting the following error you can't do queries at all 
```{
  "message": "Cannot use 'in' operator to search for 'subscribe' in null",
  "stack": "TypeError: Cannot use 'in' operator to search for 'subscribe' in null\n    at M (http://localhost:8080/html/portlet/ext/graphql/graphiql.min.js:1:583697)\n    at http://localhost:8080/html/portlet/ext/graphql/graphiql.min.js:1:580605\n    at async Object.onRun (http://localhost:8080/html/portlet/ext/graphql/graphiql.min.js:1:561142)"
}
``` 

**Expected behavior**
We need to be able to use this tool without problems 

**Desktop (please complete the following information):**
Tested on release-22.04 // Docker // FF

**Comment by freddyucv** [comment]
it is just a cache issue, you just need to clean the cache, it is working now

<img width="1539" alt="Screen Shot 2022-04-12 at 13 48 16" src="https://user-images.githubusercontent.com/10408015/163046373-89bbda05-9e21-4007-88fa-29c7c5b9205b.png">
TG8KV/screen_shot_2022-04-12_at_13.48.16.png?origin_team=T028R2630&origin_channel=D6G1WD90S

**Comment by bryanboza** [comment]
I think we should auto clean the cache after running the task b/c we will have this problem after an upgrade. Also, we need to update the starter since I'm able to reproduce starting the last docker image.

**Comment by wezell** [comment]
Agree, we should do a cache flush at the end of the startup tasks if any have run.  Maybe a new card?

**Comment by fmontes** [comment]
@wezell will that be a blocker?

**Comment by dsilvam** [comment]
Passed Internal QA. GraphQL working in 22.04 after upgrading. Tested upgrading from 22.03.

**Comment by bryanboza** [comment]
Fixed, tested on release-22.04 // Docker // FF
